src/app.js

```diff
@@ -297,6 +298,17 @@ export const start = async args => {
     screen.spawn('git', ['log', args.join('/'), ...gitLogArguments]);
   });
 
+  branchTable.key('y', async function() {
+    const selection = parseSelection(this.items[this.selected].content);
+    const gitBranch = selection[2];
+    try {
+      await copyToClipBoard(gitBranch);
+      logger.log(`Branch name ${gitBranch} copied.`);
+    } catch (error) {
+      logger.error(`Unable to copy. ${JSON.stringify(error)}`);
```

Comment:
Can we swap the period for a colon here? I think that should be fine.

Reply:
thanks for the review @jwu910  @kienD , I've modified suggested changes, plz review.